tt_metal/include/compute_kernel_api/tilize.h

```diff
@@ -40,10 +40,11 @@ ALWI void tilize_init(uint32_t icb, uint32_t block, uint32_t ocb = 16)
 /**
  * Initialize the tilize operation. To be called once at beginning of a kernel.
  */
+template<bool neginf_srcA = false, bool zero_srcA_reduce = false>
```

Comment:
Hey @mywoodstock since `neginf_srcA` was true by default before, this change will cause failures. Please set it to true by default